stdlib/2and3/wsgiref/types.pyi

```diff
@@ -1,44 +1,3 @@
-# Type declaration for a WSGI Function
-#
-# wsgiref/types.py doesn't exist and neither do the types defined in this
-# file. They are provided for type checking purposes.
-#
-# This means you cannot simply import wsgiref.types in your code. Instead,
-# use the `TYPE_CHECKING` flag from the typing module:
-#
-#   from typing import TYPE_CHECKING
-#
-#   if TYPE_CHECKING:
-#       from wsgiref.types import WSGIApplication
-#
-# This import is now only taken into account by the type checker. Consequently,
-# you need to use 'WSGIApplication' and not simply WSGIApplication when type
-# hinting your code.  Otherwise Python will raise NameErrors.
+# Obsolete, use _typeshed.wsgi directly.
```

Comment:
Why do we need to keep this file around at all? Are there usages outside typeshed?